src/main/java/frc/trigon/robot/subsystems/climber/Climber.java

```diff
@@ -0,0 +1,116 @@
+    }
+
+    @Override
+    public void updateLog(SysIdRoutineLog log) {
+        log.motor("RightClimberMotor")
+                .linearPosition(Units.Meters.of(rightMotor.getSignal(TalonFXSignal.POSITION)))
+                .linearVelocity(Units.MetersPerSecond.of(rightMotor.getSignal(TalonFXSignal.VELOCITY)))
+                .voltage(Units.Volts.of(rightMotor.getSignal(TalonFXSignal.MOTOR_VOLTAGE)));
+        log.motor("LeftClimberMotor")
+                .linearPosition(Units.Meters.of(leftMotor.getSignal(TalonFXSignal.POSITION)))
+                .linearVelocity(Units.MetersPerSecond.of(leftMotor.getSignal(TalonFXSignal.VELOCITY)))
+                .voltage(Units.Volts.of(leftMotor.getSignal(TalonFXSignal.MOTOR_VOLTAGE)));
+    }
+
+    @Override
+    public void setBrake(boolean brake) {
+        rightMotor.setBrake(brake);
+        leftMotor.setBrake(brake);
+    }
+
+    @Override
+    public SysIdRoutine.Config getSysIdConfig() {
+        return ClimberConstants.SYSID_CONFIG;
+    }
+
+    @Override
+    public void stop() {
+        rightMotor.stopMotor();
+        leftMotor.stopMotor();
+    }
+
+    void rightMotorDrive(Measure<Voltage> voltageMeasure) {
+        rightMotor.setControl(voltageRequest.withOutput(voltageMeasure.in(Units.Volts)));
+    }
+
+    void leftMotorDrive(Measure<Voltage> voltageMeasure) {
+        leftMotor.setControl(voltageRequest.withOutput(voltageMeasure.in(Units.Volts)));
+    }
+
+    void setTargetState(ClimberConstants.ClimberState targetState) {
+        currentState = targetState;
+        setTargetPosition(targetState.positionMeters, targetState.positionMeters, targetState.affectedByRobotWeight);
+    }
+
+    void setTargetPosition(double targetRightPositionMeters, double targetLeftPositionMeters, boolean affectedByRobotWeight) {
+        rightMotor.setControl(determineRequest(affectedByRobotWeight)
+                .withPosition(targetRightPositionMeters)
+                .withFeedForward(calculateFeedforward(rightMotor, affectedByRobotWeight)));
+        leftMotor.setControl(determineRequest(affectedByRobotWeight)
+                .withPosition(targetLeftPositionMeters)
+                .withFeedForward(calculateFeedforward(leftMotor, affectedByRobotWeight)));
+    }
+
+    private DynamicMotionMagicVoltage determineRequest(boolean affectedByRobotWeight) {
+        return affectedByRobotWeight ? climbingPositionRequest : nonClimbingPositionRequest;
+    }
+
+    private double calculateFeedforward(TalonFXMotor motor, boolean affectedByRobotWeight) {
+        return affectedByRobotWeight ?
+                ClimberConstants.GROUNDED_A * Math.pow(motor.getSignal(TalonFXSignal.POSITION), 2) + ClimberConstants.GROUNDED_B * motor.getSignal(TalonFXSignal.POSITION) + ClimberConstants.GROUNDED_C
+                : ClimberConstants.ON_CHAIN_A * Math.pow(motor.getSignal(TalonFXSignal.POSITION), 2) + ClimberConstants.ON_CHAIN_B * motor.getSignal(TalonFXSignal.POSITION) + ClimberConstants.ON_CHAIN_C;
+    }
```

Comment:
1. Separate the parabola calc to a method that accepts a b c and x. 
2. Accept the position as a param, not the motor. 
3. Call this function `calculateStayingInPlaceVoltage`. (BTW, this is actually feedback since this is using the current position, and not target position)